great_expectations/datasource/fluent/databricks_sql_datasource.py

```diff
@@ -21,16 +22,16 @@
     from great_expectations.compatibility import sqlalchemy
 
 
-def _parse_http_path_from_query(query: str) -> str | None:
+def _parse_param_from_query_string(param: str, query: str) -> str | None:
```

Comment:
Make method generic so it works with all query params (we now have 3 required ones)